point_of_sale/advanced_pricing_features/pricelists.rst

```diff
@@ -0,0 +1,46 @@
+=================================
+Using Pricelists in Point of Sale
+=================================
+
+You probably know the concept of happy hour. During a certain period of
+time, the barman gives a discount on some drinks (usually 50% off or a
+buy one get one free). When the period is over, everything goes back to
+normal. But what is the link with Odoo?
+
+In Odoo, you can set up happy hours. It’s one of the many possible uses
+of *Pricelists*. Those *Pricelists* will allow the creation of
+multiple prices for the same product: a regular one and a special one
+for happy hours. Available in the *PoS* app, those are really
+convenient.
```

Comment:
```suggestion
You probably know the concept of happy hour: during a certain period of
time, the bar gives discounts on drinks (usually 50% off or a
buy one get one free). When the time is over, prices go back to
normal. But how does that relate with Odoo?
Odoo allows the creation of pricelists, which could be used to set up happy hours times. 
*Pricelists* allow the creation of multiple prices for the same product: a regular one and a special one
for happy hours, for example, and they can be used in the *PoS* app.
```

Reply:
I changed the first part based on your suggestion. I made only some changes in the second part, not everything ;)